### ISSUE TITLE
Add loop one to Loop button

### ISSUE THREAD

**Issue by sayomelu** [iina/iina#2162]
When clicking the loop button (Picture below), IINA should change loop state between Loop Off/All/One, especially in Music Mode. (like iTunes dose)
(Although it has File Loop in the Menu Bar, it is hard to find, and the loop one is often used.)
<img width="412" alt="screen shot 2019-01-01 at 23 37 14" src="https://user-images.githubusercontent.com/16492117/50574194-49b64300-0e1e-11e9-8eae-7c533445ca24.png">

In the Menu Bar, the Loop should also change to the iTunes way. (Picture below)
<img width="306" alt="screen shot 2019-01-01 at 23 34 39" src="https://user-images.githubusercontent.com/16492117/50574170-eb896000-0e1d-11e9-8357-40ed67c43a04.png">

**Comment by shikhir-arora** [comment]
This would be very useful indeed 👍

**Comment by CheerLin** [comment]
yes, i was wondering why the loop button not able to loop just one file...

**Comment by falconws** [comment]
Thanks for this post.

> yes, i was wondering why the loop button not able to loop just one file...

Me TOOOOO
